### PR TITLE
Clarity Edits

### PR DESCRIPTION
**Pull request by trjordan**
A grab bag of edits to various articles to improve their clarity.

### CODE REVIEW

File: _articles/automatic-retries.md [hunk 1]
```diff
@@ -37,23 +37,31 @@ Discovery Service (RDS).
...
+overall request timeout, including the initial request and all retries.
 
-The *per_try_request_timeout* field sets a **timeout for each retry**. Without
-this parameter, any request that times out will not be retried, since the
-default is the same as the calling request’s timeout. While it’s not a big deal
-to leave this out, setting it to the 99th percentile of normal latency allows
-Envoy to retry requests that are taking a long time due to a failure. (Note
-that this limit may be longer than the total request timeout — more on that
-below.)
+The `per_try_request_timeout` field sets a **timeout for each retry** in
```
Comment (9len) [inline _articles/automatic-retries.md]: I think it may be in nanos

Comment (trjordan) [reply _articles/automatic-retries.md]: jeez, this is just wrong. The name is `per_try_timeout_ms`.

File: _articles/automatic-retries.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ issues.
 Envoy provides a simple configuration option for retrying requests. Consider
 specifics of your system as you set up retries across each route:
 
-*Choose appropriate defaults
-*Limit retry-able requests
-*Consider the calling context
+* Choose appropriate defaults
```
[the remark this answers is not in the document]
Comment (trjordan) [reply _articles/automatic-retries.md]: Let's! Did a repo-wide find/replace.

File: _articles/service-discovery.md [hunk 3]
```diff
@@ -32,14 +32,14 @@ first step in setting up your control plane is connecting your service
 discovery. This is generally broken down into three steps:
 
 1. Decide on a control plane implementation
-2. Mirror service definitions to Envoy clusters
-3. Mirror hosts/containers/instances to Envoy endpoints
+2. Public service definitions to Envoy clusters
```
[the remark this answers is not in the document]
Comment (trjordan) [reply _articles/service-discovery.md]: Whoops, yep.

**Pull request merged**